src/browser/rvm/rvm_message_bus.ts

```diff
@@ -104,6 +105,11 @@ interface DownloadRuntimeMsg extends RvmMsgBase {
 
 }
 
+export interface AppCloseRequestedOptions {
+    uuid: string;
+    sourceUrl: string;
```

Comment:
Is sourceUrl needed?

Reply:
The RVM uses sourceUrl as it's identifier, so we will need to send it yes.